spec/robots/dor_repo/gis_assembly/generate_tag_spec.rb

```diff
@@ -11,7 +11,7 @@
     allow(LyberCore::WorkflowClientFactory).to receive(:build).and_return(workflow_client)
   end
 
-  describe '#perform' do
+  describe '#perform_work' do
     let(:object_client) do
```

Comment:
nitpick -- the robot method is called `perform_work`, so fixed in other specs when I noticed this